Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build Stage
-FROM --platform=$BUILDPLATFORM golang:1.19-alpine as builder
+FROM --platform=$BUILDPLATFORM golang:1.21-alpine as builder
```

Comment:
If upgrading to go 1.21, there are a few other places that need to be changed. Please add the other changes needed for go 1.21, or remove this and make a separate PR for upgrading to go 1.21.

Reply:
made a separate PR to bump golang version